incubator/patroni/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: patroni
 description: "Highly available elephant herd: HA PostgreSQL cluster."
-version: 0.2.0
+version: 0.2.1
```

Comment:
@alexeyklyukin I'm wondering should we use the spilo version (1.0-p5 etc., 1.2-p8, ...) or the patroni version (1.2.5)?